scene/main/node.cpp

```diff
@@ -3381,6 +3362,11 @@ void Node::_validate_property(PropertyInfo &p_property) const {
 	}
 }
 
+String Node::_to_string() {
+	ERR_THREAD_GUARD_V(String());
```

Comment:
I wonder why this exists. None other class has a guard in this method.

Reply:
IN all likelihood Reduz copied and pasted it back when he wanted to make Nodes as thread-safe as possible when implementing Thread Groups.